### PR TITLE
[release-1.11] oci: Don't use created from runc state

### PR DESCRIPTION
**Pull request by mrunalp**
@runcom PTAL
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

File: oci/container.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ type ContainerVolume struct {
 // ContainerState represents the status of a container.
 type ContainerState struct {
 	specs.State
-	Created   time.Time `json:"created"`
+	Created   time.Time `json:"createdat"`
```
Comment (runcom) [inline oci/container.go]: I'd rather use camelCase as for exitCode below

**Comment by runcom** [comment]
looks good except a nit, we can merge afterwards

**Comment by mrunalp** [comment]
yeah we need to provide backwards compatibility.

**Comment by umohnani8** [comment]
@mrunalp do we still need this?

**Comment by mrunalp** [comment]
Closing as we fixed this a different way.

**Pull request closed without merging**